app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -503,7 +516,17 @@ fun CreateActivityScreen(
               }
               Spacer(modifier = Modifier.height(LARGE_PADDING.dp))
               Button(
-                  enabled = title.isNotEmpty() && description.isNotEmpty(),
+                  enabled =
+                      title.isNotEmpty() &&
+                          description.isNotEmpty() &&
+                          price.isNotEmpty() &&
+                          placesMax.isNotEmpty() &&
+                          selectedLocation != null &&
+                          selectedOptionType != "Select a type" &&
+                          selectedOptionCategory != "Select a category" &&
+                          startTime.isNotEmpty() &&
+                          duration.isNotEmpty() &&
+                          dueDate.toDate().after(Timestamp.now().toDate()),
```

Comment:
very good, these missing conditions were painful because it was not very coherent when adding/editing